CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2017-2018, Intel Corporation
+# Copyright 2017-2023, Intel Corporation
```

Comment:
this date update is suppose to be done in the first commit, not in the second one